tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Services/ProfileServiceTests.cs

```diff
@@ -32,14 +32,24 @@ public async Task GetProfileDataAsync_GivenAnApplicationUserExistsWithOrganisati
                 .WithOrganisationFunction(OrganisationFunction.Authority)
                 .Build();
 
+            var expectedOrganisation = OrganisationBuilder
+                .Create()
+                .WithName("Primary Health Trust")
```

Comment:
If the `OrganisationBuilder` has a sensible default for the name, which I think it does, do you think the `WithName` could be omitted here, to simplify the code? This also applies to the rest of the file.